Admin/AdminBuilder.php

```diff
@@ -94,7 +94,7 @@ public function getEditForm(AdminInterface $admin)
     public function getShowForm(AdminInterface $admin)
     {
         $show = new FieldDescriptionCollection();
-        $mapper = new ShowMapper($admin->getShowBuilder(), $show, $admin);
+        $mapper = new ShowMapper($this->showBuilder, $show, $admin);
```

Comment:
I did change this because If not, the constructor of adminbuilder makes no sense, it is not using the arguments